cmd/es-rollover/app/flags.go

```diff
@@ -51,6 +53,8 @@ func AddFlags(flags *flag.FlagSet) {
 	flags.Bool(useILM, false, "Use ILM to manage jaeger indices")
 	flags.String(ilmPolicyName, "jaeger-ilm-policy", "The name of the ILM policy to use if ILM is active")
 	flags.Int(timeout, 120, "Number of seconds to wait for master node response")
+	flags.Bool(dependenciesEnabled, false, "Enable dependency index rollover")
```

Comment:
this doesn't seem like a backwards compatible change. Maybe you want to change the flag to "skip-dependencies", where default value false would retain the existing behavior.

Reply:
Note: this only applies to the CLI flag, in the rest of the code it's fine to use `dependenciesEnabled` since the default value does not play a role there when the param is passed explicitly.

Reply:
Fair point, I'll do the change

Reply:
On the other hand, having this feature enable is indeed breaking some old jaeger installations.